src/undofeature.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+import Feature from '../feature.js';
+import UndoCommand from './undocommand.js';
+
+/**
+ * Undo feature.
+ *
+ * Undo features brings in possibility to undo and re-do changes done in Tree Model by deltas through Batch API.
+ */
+export default class UndoFeature extends Feature {
```

Comment:
The idea is that if a features is called like the whole package it's enough to say `'undo'` to enable it (the name is extended automatically to `undo/undo`).

As for collisions, all other Bold related things must be named differently. Command will be `BoldCommand`, we may have `BoldSchema` or whatever else comes to our minds. The entry point of such a package is the feature so all other names have to be adjusted.